main.py

```diff
@@ -2,6 +2,53 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    create_genre = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+
+    for name in create_genre:
+        Genre.objects.create(
+            name=name
+        )
+
+    create_actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
```

Comment:
it is bad name for variables here, could u just name it like genres and actors ?